compiler/testData/codegen/bytecodeText/sam/samWrapperForNullableInitialization.kt

```diff
@@ -17,8 +16,16 @@ fun test() {
     JFoo.foo2({}, runnable())
 }
 
+// JVM_TEMPLATES
 // @TestKt.class:
```

Comment:
Minor:
```
// @TestKt.class
// 2 NEW
``` could be extracted in common part

Reply:
Done